src/Prism.Plugin.Logging.Common/Http/HttpLogger.cs

```diff
@@ -7,12 +7,11 @@ namespace Prism.Logging.Http
 {
     public class HttpLogger
     {
+        private static HttpClient _client = new HttpClient();
```

Comment:
I'd like to avoid a static reference like this. It would be better to have this as a protected field, and implement `IDisposable` to dispose the HttpClient